src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -141,6 +141,7 @@ const SettingsModalStackNavigator = createModalStackNavigator({
     Settings_Lounge_Access: () => require('../../../pages/settings/Profile/LoungeAccessPage').default,
     Settings_Wallet: () => require('../../../pages/settings/Wallet/WalletPage').default,
     Settings_Wallet_DomainCards: () => require('../../../pages/settings/Wallet/ExpensifyCardPage').default,
+    Settings_Report_Fraud: () => require('../../../pages/settings/Wallet/ReportFraudPage').default,
```

Comment:
Could we make this `Settings_Wallet_ReportVirtualCardFraud` for consistency?

Reply:
Sure, done!